### ISSUE TITLE
TypeError: clns is not iterable (cannot read property undefined)

### ISSUE THREAD

**Issue by Frydac** [kucherenko/jscpd#402]
**Describe the bug**
When running `jscpd .` in some folders I get the error message below

**To Reproduce**
I cannot share the content of the folders, but it is just a folder with a bunch of C .c and .h files, no subdirictories. It doesn't happen for all folders, only sometimes. (if really needed I can maybe bisect the content untill I can reproduce it without breaking my nda)
```
▶ jscpd .
TypeError: clns is not iterable (cannot read property undefined)
    at /home/emile/.node_modules/lib/node_modules/jscpd/node_modules/@jscpd/finder/dist/in-files-detector.js:50:24
```

When I change that line to:
```
                    if (typeof clns !== 'undefined') {
                        clones.push(...clns);
                    }
```
then it seems to work around the issue at least, although I'm not sure if this leads to the intended behavior. I have no clue what clns nor clones should represent.

**Desktop (please complete the following information):**
 - OS: Manjaro linux (rolling release, recently updated)
```
▶ node --version                                                                                         
v14.10.0
▶ npm --version       
6.14.8
▶ jscpd --version                                               
3.3.19
```

**Comment by kucherenko** [comment]
Thank you for the report, will fix it

**Comment by knom** [comment]
I run into this right now and am blocked :-(

Any workaround?
I'm also happy to do a quick bugfix?

Thx.

**Comment by kucherenko** [comment]
fixed in jscpd@3.3.21
